site/static/js/main.js

```diff
@@ -427,6 +427,39 @@ function toggleMobileMenu(icon){
   }
 }
 
+// Checks if the subnav is overflowing
+function subnavOverflowing() {
+  var subnav = document.querySelector('.sub-nav');
+  if (!subnav) return false;
+
+  var padding = 0;
+  padding += parseFloat(window.getComputedStyle(subnav, null).getPropertyValue('padding-left'));
+  padding += parseFloat(window.getComputedStyle(subnav, null).getPropertyValue('padding-right'));
+
+  var childrenWidth = 0;
+  for (var i = 0; i < subnav.children.length; i++) {
+    childrenWidth += subnav.children[i].offsetWidth;
+  }
+
+  return childrenWidth > window.innerWidth - padding;
+}
+
+function togglePageTopPadding() {
+  var navItem = document.querySelector('.sub-nav .nav-item');
+  if (!navItem) return;
+
+  var page = document.querySelector('.page');
+  if (!page) return;
+
+  if (subnavOverflowing()) {
+    if (page.hasAttribute('style')) return;
```

Comment:
This implies that we are only supporting a single resize?

Reply:
This prevents the padding from growing 100s of times per resize.  I realise that this check being indirect isn't a great way.  Maybe I could hold onto the page padding value in a global variable instead?